src/components/__tests__/ContextProvider-test.tsx

```diff
@@ -21,7 +22,7 @@ describe("ContextProvider", () => {
     let context = instance.getChildContext();
     expect(context.store.getState().libraries).to.be.ok;
     expect(context.store.getState().library).to.be.ok;
-    expect(context.pathFor).to.equal(instance.pathFor);
```

Comment:
So, `pathFor` is actually not used in this app. It's not in the context so it's undefined, and it's not in the instance so that is undefined too. That's why this test passed but it should not be checked. Instead, I passed down `username` which does get added to the context.